Error handling for movies and people

wojtekkkwojtek opened the pull request:
Hi!

Currently I've been trying to implement "ErrorMessage" component to index.js files for movies and people. Something went wrong literally 😂. Do you have any idea what is going there at the moment of rendering? 
PS: the URL address is changed deliberately for testing and the branch isn't ready to be merged.

miloszzaj commented:
I think the problem is in styled component - wrong Link usage  and problem with props which pass color information. What in more, you didn't write correct Link path in index.

wojtekkkwojtek commented:
> I think the problem is in styled component - wrong Link usage and problem with props which pass color information. What in more, you didn't write correct Link path in index.

Your suggestions were very useful! I made several errors which I already fixed. The first problem was Link implementation to styled.js and the button was styled incorrectly. I imported {Link} and {toMovie} into index.js and added the path to the button. I also changed yield call -> put in saga file. 
In order to present you'll the working solution (Error rendering), I decided to not correct the typo in the URL in src/common/assets/generalData/fetchedData.js. 

Please, take a look once again.

IT2021db commented on src/features/movies/MovieList/movieListSaga.js:
```diff
@@ -11,7 +11,7 @@ function* fetchMovieListHandler() {
     } catch (error) {
-        yield call(fetchMovieListError())
+        yield put(fetchMovieListError())
         yield call(alert, 'cos nie tak z listy')
```
I suggest writing error/alert info in English so the browser is dedicated to everyone.

IT2021db reviewed (approved):
There's lack of semicolons in the end of lines. I it's important.

IT2021db commented on src/common/components/ErrorMessage/styled.js:
```diff
@@ -35,14 +34,14 @@ export const Announcement = styled.p`
-  color: ${({ theme }) => theme.colors.white};
-  background-color: ${({ theme }) => theme.colors.blue};
+  color: ${({ theme }) => theme.color.white};
+  background-color: ${({ theme }) => theme.color.blue};
```
There's a mistake. In this line should be: `theme.colors.blue`

IT2021db commented:
In general, very good work!
